content/docs/rfcs/10/README.md

```diff
@@ -61,177 +61,184 @@ These protocols and their semantics are elaborated on in their own specs.
 that the encoding of data field is protobuf.
 
-The default PubSub topic SHOULD be used for all protocols. This ensures a
-connected network, as well some degree of metadata protection. It MAY be
-different if or when:
-<!-- TODO discuss anonymity related to one pubsub topic -->
+The default PubSub topic SHOULD be used for all protocols.
+This ensures a connected network, as well some degree of metadata protection.
+It MAY be different if or when:
 - Different applications have different message volume
 - Topic sharding is introduced
 - Encoding is changed
 - Version is changed
 
 ## Discovery domain
 
-Discovery domain is not yet implemented. Currently static nodes should be used.
+The discovery domain is not yet specified.
+As such, currently static nodes should be used.
+Please refer to specific implementations for details on publicly available nodes, as these may change over time.
 
-<!-- TODO: To document how we use Discovery v5, etc. See https://github.com/vacp2p/specs/issues/167 -->
+<!-- TODO: To document how we use Discovery v5, EIP-1459 (DNS- based) etc. -->
 
 ## Request/reply domain
 
-This consists of two main protocols. They are used in order to get Waku to run
-in resource restricted environments, such as low bandwidth or being mostly
-offline.
+This consists of two main protocols.
+They are used in order to get Waku to run in resource restricted environments,
+such as low bandwidth or being mostly offline.
 
 ### Historical message support
 
 **Protocol identifier***: `/vac/waku/store/2.0.0-beta1`
 
-See [13/WAKU-STORE](/spec/13) spec for more details.
+See [13/WAKU2-STORE](/spec/13) spec for more details.
 
 ### Content filtering
 
 **Protocol identifier***: `/vac/waku/filter/2.0.0-beta1`
 
-See [14/WAKU-FILTER](/spec/14) spec for more details.
-
+See [14/WAKU2-FILTER](/spec/14) spec for more details.
 
 # Upgradability and Compatibility
 
 ## Compatibility with Waku v1
 
-Waku v2 and Waku v1 are different protocols all together. They use a different
-transport protocol underneath; Waku v1 is devp2p RLPx based while Waku v2 uses
-libp2p. The protocols themselves also differ as does their data format.
-Compatibility can be achieved only by using a bridge that not only talks both
-devp2p RLPx and libp2p, but that also transfers (partially) the content of a
-packet from one version to the other.
-
-See [bridge spec](waku-bridge.md) for details on a bridge mode.
+Waku v1 and Waku v2 are different protocols all together.
+They use a different transport protocol underneath; Waku v1 is devp2p RLPx based while Waku v2 uses libp2p.
+The protocols themselves also differ as does their data format.
+Compatibility can be achieved only by using a bridge that not only talks both devp2p RLPx and libp2p, but that also transfers (partially) the content of a packet from one version to the other.
 
+See [15/WAKU-BRIDGE](/spec/15) for details on a bidirectional bridge mode.
```

Comment:
This link does not work (likewise for the links to other specs)

Reply:
It should work once it is live, e,g http://rfc.vac.dev/spec/15